rules/cross-platform/execution_python_script_in_cmdline.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2021/01/13"
+maturity = "production"
+updated_date = "2021/01/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when a Python script is executed from the commandline input. Attackers often use this method to execute
+malicious scripts and avoiding writing it to disk.
+"""
+false_positives = ["Legitimate Python scripting activity."]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Suspicious Python Script Execution via the CommandLine"
+risk_score = 47
+rule_id = "ee9f08dc-cf80-4124-94ae-08c405f059ae"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "macOS", "Windows", "Threat Detection", "Execution"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
+ process.name:(Python* or python*) and process.args:("-c" and import*sys,*)
+'''
+
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1059"
+name = "Command and Scripting Interpreter"
+reference = "https://attack.mitre.org/techniques/T1059/"
+
```

Comment:
could add the subtechnique for python coverage https://attack.mitre.org/techniques/T1059/006/